tfx/dsl/component/experimental/decorators_test.py

```diff
@@ -140,8 +140,10 @@ def verify_beam_pipeline_arg(a: int) -> OutputDict(b=float):  # pytype: disable=
 
 def verify_beam_pipeline_arg_non_none_default_value(
     a: int,
-    beam_pipeline: BeamComponentParameter[beam.Pipeline] = beam.Pipeline(),
+    beam_pipeline: BeamComponentParameter[beam.Pipeline] = 0,
```

Comment:
`beam_pipeline: BeamComponentParameter[beam.Pipeline] = None` looks better

Reply:
Oh I see. The typing error is pretty clear. You cannot assign an integer to other type.

If it is to avoid B008, then how's about doing it in this way?

```python
_TestBeamPipelineArgs = ['--my_testing_beam_pipeline_args=foo']
_TestBeamEmptyPipeline = beam.Pipeline()  # <- Newly added

...

def verify_beam_pipeline_arg_non_none_default_value(
    a: int,
    beam_pipeline: BeamComponentParameter[beam.Pipeline] = _TestBeamEmptyPipeline,
) -> OutputDict(b=float):  # pytype: disable=invalid-annotation,wrong-arg-types
  del beam_pipeline
  return {'b': float(a)}
```

Reply:
Thanks! That works. I have updated.